.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
 
             - name: Install dependencies
               run: |
-                  python -m pip install pre-commit==2.11.1
+                  python -m pip install pre-commit==3.0.3
```

Comment:
This need to be compatible with the pre-commit.yaml, I guess you change the to fix the pre-commit issue. If it is not work, please revert it back to old version.